### PR TITLE
THE HULK GENE IS REAAAAAAAAAAAAAAL

### PR DESCRIPTION
**Pull request by Yarinoi**
can we get much higher
so high~

dont get excited powergamers its traitor only and now antag gear

# Document the changes in your pull request
Dr. Bruce B. was SIC's lead scientist in gamma radiation. One morning, after a particularly heavy workload, he snapped, and injected himself with a mysterious syringe. Suddenly, he underwent rapid muscle growth, and green skin pigmentation, and he began to slaughter everyone in sight with his bare hands, from employee to security personnel, before finally being killed by an officer choosing to use lethal weaponry against him.
This incident interested Vahlen Pharmaceuticals. After sending a team to pick up Dr. Bruce B's research, the syringe, and anything that could prove of use, they reverse-engineered the serum that he injected himself with, and dubbed it the Hulk Mutator.
And now YOU can have it for the low low price of 75% of your TC!!

Adds a human-only purchasable hulk mutator for 15 TC. This PR will not be controversial whatsoever.
Hulks no longer scream over comms.
Hulk can no longer be distributed to others.
Hulk is now mutadone immune so that chemists cant make traitors burn 15 tc with one simple trick.

# Why is this good for the game?
Hulk is a very fun and silly gene. It being removed was an unfortunate sacrifice, as total antistun and wallbusting is way too stupid powerful for crew to have. So I gave it this.
Also humans now get their very own species restricted item. Good for them.

# Testing
Actually tested this time, it... can be bought. And does dispense a hulk mutator.
Hulk genetic sequence corruption _does_ work.. sorta. It permanently makes the DNA console unusable, so I'll have to figure out how to fix that.

# Spriting
N/A

# Wiki Documentation
Add the item to the Syndicate Items wiki, and possibly to Gene under some hidden syndicate mutations thing.

# Changelog

:cl:  
rscadd: HULK IS NOW BUYABLE BY HUMAN TRAITOR FOR 15 TC!!!
tweak: MINOR CHANGES TO HULK!!!
/:cl:

### CODE REVIEW

**Comment by Yarinoi** [comment]
wait shit

**Comment by Yarinoi** [comment]
ok we golden

**Comment by bruhlookatthisdood** [comment]
image

**Comment by bruhlookatthisdood** [comment]
buff hulk

**Comment by KonKonBreezeFox** [comment]
who will win, the traitor that spent 15 tc on hulk, or the chemist with mutadone spray

**Comment by KingSuperBOB** [comment]
I dont think its worth 15 TC considering its a mutator so you would have a hard time reinforcing it without help and I doubt its metashielded but hey discounts exist

**Comment by Marmio64** [comment]
yeah i'd rather get sleeping carp for 1 tc cheaper. Hulk gives you stunproofing sure, wall punches, and powerful punches but hulk being removable and extremely obvious makes it pretty weak in comparison. Should be in the realm of 10-12 TC at most.

**Comment by Therandomhoboo** [comment]
There was a reason why Hulk was removed. Free anti-stun, able to punch down walls and more. Regardless of being able to be removed by mutadone this isn't a good idea.

**Comment by KingSuperBOB** [comment]
> There was a reason why Hulk was removed. Free anti-stun, able to punch down walls and more. Regardless of being able to be removed by mutadone this isn't a good idea.

This is a mutator that only works for one person and cant be printed I do not see the issue

**Comment by MajManatee** [comment]
have you tried seeing if you can save it to a gene console and distribute it?

**Comment by Therandomhoboo** [comment]
> > There was a reason why Hulk was removed. Free anti-stun, able to punch down walls and more. Regardless of being able to be removed by mutadone this isn't a good idea.
> 
> This is a mutator that only works for one person and cant be printed I do not see the issue

Re-read what I said.

**Comment by Yarinoi** [comment]
> yeah i'd rather get sleeping carp for 1 tc cheaper. Hulk gives you stunproofing sure, wall punches, and powerful punches but hulk being removable and extremely obvious makes it pretty weak in comparison. Should be in the realm of 10-12 TC at most.

> I dont think its worth 15 TC considering its a mutator so you would have a hard time reinforcing it without help and I doubt its metashielded but hey discounts exist

> who will win, the traitor that spent 15 tc on hulk, or the chemist with mutadone spray

fair enough BUT people see hulk and immediately they think 'this is an overpowered thing', and besides, it's been a while (that i know of) since we've last had it. i might willing to bump it down to 14 tc, though, if anyone can give me a convincing enough reason to do so, but i'll go into why that might be a bad idea

> have you tried seeing if you can save it to a gene console and distribute it?

valid worry. it'd make my day if this wasnt the case but if it can be distributed, i've been considering turning it from just a mutator into an item that just gives you hulk on activate, which cant be used on other people. alas i lack coding skill so that might take some time
my real fear with distribution is op antag teamups, less so with distribution to crew (remember that this change would make hulk syndie tech/weaponry!) because 1 hulk is probably managable, but noone wants to deal with 3 or 4 hulks with antag gear. this is also why i made hulk 15 tc, because if it was 14 tc, that leaves you with 6 tc that can be used to purchase an emag, and thus give you access to tc bounties. and with hulk's ability to smash down walls, getting the items for those bounties will be a piece of cake. this was the same reason why holopara cost was increased to 15 tc, people

tldr hulk with other higher-cost antag gear is bad
anyways off to see if hulk can be distributed

**Comment by Yarinoi** [comment]
fuck
image
(this is hulk)

could there be a way to make it unsaveable?
edit: might have found a way, will see if it works

**Comment by Yarinoi** [comment]
mapbot why arent you compiling i didnt even do any mapping

**Comment by Yarinoi** [comment]
> being able to be removed by mutadone

to be honest it might need a BUFF in that department by starting out reinforced, buster arm and holopara would both be way less strong if they could be removed by just spraying a certain, pretty easy to make chem on the user

**Comment by Yarinoi** [comment]
> have you tried seeing if you can save it to a gene console and distribute it?

this has now been partially addressed, hulk now corrupts a person's genetic sequence similar to monkified, but it has a strange bug where it makes the console unusable by perpetually showing the 'genetic sequence unreadable' message, even when noone's in it. that or it only happens on scramble. in which case eeeeeeeeeeeeh

**Comment by Therandomhoboo** [comment]
So now Hulk is mutadone proof, we are back at the reason why Hulk was removed in the first place? 

Can we just skip the whole merge and obvious redflag and inevitable revert and just not?

**Comment by Tenn0x** [comment]
> So now Hulk is mutadone proof, we are back at the reason why Hulk was removed in the first place?
> 
> Can we just skip the whole merge and obvious redflag and inevitable revert and just not?

Traitor item and with a 15 TC cost

**Comment by DeadlyBag** [comment]
Used to play when hulk was active and the issue wasnt the stunproofing or wallbreak on a singular person
the issue was the army of greytiders/validhunters having it.

Dealing with one hulk? pretty easy
Dealing with a dozen hulks? not easy

Not being able to distribute it fixes the issue and for 15 TC this seems fair considering you lose all ranged gun capabilities

**Comment by Yarinoi** [comment]
tried to use allow_transfer = FALSE at the suggestion of molti, but that mechanic seems to be broken right now, sooo.. shit
going to see if that's fully bugged or if it's just hulk for some reason

**Comment by Yarinoi** [comment]
yeah it does look bugged, dont merge until either that's fixed or i find a solution

**Review by ToasterBiome — APPROVED** [review]

**Comment by Moltijoe** [comment]
ah, didn't read comments, blame biome for not slapping a do not merge label on this pr

**Comment by Yarinoi** [comment]
>please dont merge until the allow_transfer bug is fixed
>*merges anyway*

oh well, field testing i guess
